frame/support/test/tests/pallet_ui.rs

```diff
@@ -31,6 +31,6 @@ fn pallet_ui() {
 	std::env::set_var("RUSTFLAGS", "--deny warnings");
 
 	let t = trybuild::TestCases::new();
-	t.compile_fail("tests/pallet_ui/*.rs");
+	t.compile_fail("tests/pallet_ui/* */.rs");
```

Comment:
I think this is unintended?

Reply:
blame shows @kianenigma added it yesterday. I mean it's not bad, it just means sub-directories would work too, so I'm OK with it if this was intended.

Reply:
was indeed a mistake though. Will revert, thanks for the catch @KiChjang